src/lib/expression/between_expression.cpp

```diff
@@ -6,27 +6,48 @@
 
 namespace opossum {
 
+static PredicateCondition _get_between_predicate_expression(bool left_inclusive, bool right_inclusive) {
```

Comment:
btw, should left_inclusive and right_inclusive be renamed to lower_inclusive and upper_inclusive?